### DISCUSSION TITLE
Jump to documentation window

### DISCUSSION THREAD

**Discussion by IamSmeagol** [hrsh7th/nvim-cmp#1517] [Q&A] [answered]
When using the lua_lsp, some functions provide links to documentation. Is it possible to jump to the window of documentation? If so, how can I add it to my configuration. 

I could not find it in the documentation https://github.com/hrsh7th/nvim-cmp/blob/main/doc/cmp.txt#L287

Here is my current settings for cmp https://github.com/Smeagol2069/.config/blob/main/nvim/lua/smeagol/plugins/nvim-cmp/main.lua
I would like to jump to the link to easily read more about related functions
![image](https://user-images.githubusercontent.com/78649849/231508049-76c6ba72-aabe-4709-95b9-cf419111ad4f.png)

**Comment by yutkat** [comment]
These plugins can display `signature help` automatically from LSP.

https://github.com/hrsh7th/cmp-nvim-lsp-signature-help
https://github.com/ray-x/lsp_signature.nvim
https://github.com/folke/noice.nvim

**Reply by IamSmeagol** [reply]
Those seem like useful plug-ins, thank you. However I think my question was unclear, sorry. I want to know how I could access the links within the documentation window. So I can access the documentation of related functions

**Reply by yutkat** [reply, marked as answer]
You mean you want to enter the floating window?

This way you can enter the floating window.
`:lua vim.lsp.buf.signature_help()`
`<C-W>W`

You can assign this to insert mode as well.
`inoremap <C-f> <C-o><C-w>W`

**Reply by IamSmeagol** [reply]
Thank you for the help! I was able to get it working (With help from [chrisgrieser](https://github.com/chrisgrieser) )
```lua
keymap("i", "<C-f>", function()
    vim.cmd.stopinsert()
    vim.lsp.buf.signature_help()
    vim.defer_fn(function() vim.cmd.wincmd("w") end, 100)
    vim.keymap.set("n", "q", ":close<CR>", { buffer = true })
end)
```
